bindings/python/pinocchio/shortcuts.py

```diff
@@ -76,6 +102,57 @@ def buildModelsFromUrdf(
     return tuple(lst)
```

Comment:
To avoid code duplication, _buildModelsFromUrdfWithoutRootJointName should call _buildModelsFromUrdfWithRootJointName.

Reply:
Why do 2 functions, in this case ? We could just add a default argument root_joint_name, no ?